packages/core/src/logic/operators.ts

```diff
@@ -1,7 +1,111 @@
-import { OperatorType, Type } from '../domain';
+import { OperatorType, Rank, Type } from '../domain';
 import { createOperator, createWorker } from './creators';
 
-export const plus = createOperator(OperatorType.PLUS, [
-  { type: Type.NUMBER, worker: createWorker((a: number, b: number) => a + b) },
-  { type: Type.STRING, worker: createWorker((a: string, b: string) => a + b) },
+const max = createOperator(OperatorType.MAX, [
+  {
+    type: Type.NUMBER,
+    worker: createWorker((a: number, b: number) => (a > b ? a : b)),
+  },
 ]);
+const min = createOperator(OperatorType.MIN, [
+  {
+    type: Type.NUMBER,
+    worker: createWorker((a: number, b: number) => (a > b ? b : a)),
+  },
+]);
+const sum = createOperator(OperatorType.SUM, [
+  {
+    type: Type.NUMBER,
+    worker: createWorker((a: number, b: number) => a + b),
+  },
+]);
+const prod = createOperator(OperatorType.PROD, [
+  {
+    type: Type.NUMBER,
+    worker: createWorker((a: number, b: number) => a * b),
+  },
+]);
+
+//- Performs a logical and across the elements.
+const land = createOperator(OperatorType.LAND, [
+  {
+    type: Type.BOOLEAN,
+    worker: createWorker((a: boolean, b: boolean) => a && b),
+  },
+]);
+
+//- Performs a logical OR across the elements.
+const lor = createOperator(OperatorType.LOR, [
+  {
+    type: Type.BOOLEAN,
+    worker: createWorker((a: boolean, b: boolean) => a || b),
+  },
+]);
+
+// Performs a bitwise and across the bits of the elements.
+const band = createOperator(OperatorType.BAND, [
+  {
+    type: Type.NUMBER,
+    worker: createWorker((a: number, b: number) => a & b),
+  },
+]);
+
+// Performs a bitwise OR across the bits of the elements.
+const bor = createOperator(OperatorType.BOR, [
+  {
+    type: Type.NUMBER,
+    worker: createWorker((a: number, b: number) => a | b),
+  },
+]);
+
+const lxor = createOperator(OperatorType.LXOR, [
+  {
+    type: Type.NUMBER,
+    worker: createWorker((a: number, b: number) => (a || b) && !(a && b)),
+  },
+]);
+
+const bxor = createOperator(OperatorType.BXOR, [
+  {
+    type: Type.NUMBER,
+    worker: createWorker((a: number, b: number) => a ^ b),
+  },
+]);
+
+interface IMaxlocObj {
+  rank: Rank;
+  value: number;
+}
```

Comment:
suggestion(blocking): IMHO, it would be better to extract this into the domain with a name like `LocationOperatorArg` or something, because now we use `IMaxloc...` in minloc operator

Reply:
Nice, thanks